cmd/notation/cert/list.go

```diff
@@ -22,6 +22,20 @@ func certListCommand(opts *certListOpts) *cobra.Command {
 		Use:     "list",
 		Aliases: []string{"ls"},
 		Short:   "List certificates in the trust store.",
+		Long: `List certificates in trust store
+
+Example - List all certificate files stored in the trust store
+  notation cert ls
```

Comment:
Can we use `notation certificate` for all the examples?

Reply:
@yizha1 Now all CLI commands are using short aliases by default. Let's keep using the short alias in each command line example. It will be easy for users to type and memorize it.